unity/Assets/Poolakey/Demo/PoolakeyExample.cs

```diff
@@ -0,0 +1,65 @@
+﻿using UnityEngine;
+using Poolakey.Scripts;
+using UnityEngine.UI;
+using Poolakey;
+using Poolakey.Scripts.Data;
+
+public class PoolakeyExample : MonoBehaviour
+{
+    public Text ConsoleText;
+
+    private Payment payment;
+    private PurchaseInfo purchase;
+
+    void Start()
+    {
+        Log("Poolakey Plugin Version: " + PluginVersion.VersionString);
+        SecurityCheck securityCheck = SecurityCheck.Disable();
+        PaymentConfiguration paymentConfiguration = new PaymentConfiguration(securityCheck);
+        payment = new Payment(paymentConfiguration);
+    }
+
+    public void Connect()
+    {
+        _ = payment.Connect();
```

Comment:
I think It's better to show what is the result of `Connect` method and don't ignore it, because it's a sample app and we can cover some simple usages of methods.